wasp/wasp.go

```diff
@@ -105,8 +105,8 @@ const (
 
 // Segment load test schedule segment
 type Segment struct {
-	From     int64
-	Duration time.Duration
+	From     int64         `json:"from"`
+	Duration time.Duration `json:"duration"`
```

Comment:
We've discussed we can add:
```
LoadType string (plain or steps, any name you choose)
Start time.Time
End time.Time 
```
Then devs can compare only plain segments as the most stable one and ignore steps/warmup

Reply:
added that, but still need to add exclusion of stepped segments